src/index.html

```diff
@@ -21,7 +21,370 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page_body">
+    <header class="header">
+      <a href="#scr/images">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="header__logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="../src/images/imac.jpeg"
+          alt="imac"
+          class="card__product-image"
+        />
+        <p class="card__product-name">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="card__product-rating">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__product-rating-review">Reviews: 5</p>
+        </div>
+        <div class="card__product-price">
+          <p class="card__product-price-text">Price:</p>
+          <p class="card__product-price-numbers">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__product-buy-link"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
```

Comment:
The code for the card is repeated several times. In a real-world scenario, you would likely use a templating engine to avoid repeating the same code. However, for this task, it's acceptable.